MdeModulePkg/Universal/PCD/Dxe/Service.c

```diff
@@ -738,19 +738,10 @@ LocateExPcdBinary (
   VOID
   )
 {
-  EFI_STATUS  Status;
-
-  //
-  // Search the External Pcd database from one section of current FFS,
-  // and read it to memory
-  //
-  Status = GetSectionFromFfs (
-             EFI_SECTION_RAW,
-             0,
-             (VOID **)&mDxePcdDbBinary,
-             &mDxePcdDbSize
-             );
-  ASSERT_EFI_ERROR (Status);
+  // MU_CHANGE begin
+  mDxePcdDbBinary = (DXE_PCD_DATABASE *)PcdDatabaseLoaderLoad (NULL);
+  ASSERT (mDxePcdDbBinary != NULL);
+  // MU_CHANGE end

```

Comment:
Should we change this to comment out the original code, the current MU_CHANGE style?

Reply:
I think so.